types/layout.d.ts

```diff
@@ -29,7 +29,8 @@ export const enum Source {
   Discover,
   Mutation,
   Scroll,
-  Input
+  Input,
+  Css
```

Comment:
>Css [](start = 2, length = 3)

Add trailing ,

Reply:
The last entry in a Javascript object shouldn't have a trailing ",". See Line 36 below for an example. 

---
In reply to: [243140562](https://github.com/Microsoft/clarity-js/pull/116#discussion_r243140562) [](ancestors = 243140562)